src/components/pages/admin/events/dashboard/reports/Index.js

```diff
@@ -31,8 +31,8 @@ class Report extends Component {
 		Bigneon()
 			.events.read({ id: eventId })
 			.then(response => {
-				const { name } = response.data;
-				this.setState({ eventName: name });
+				const { name, sales_start_date } = response.data;
```

Comment:
Just as an fyi you can redefine a variable name in one step like so:
```suggestion
				const { name, sales_start_date: salesStart } = response.data;
```

Reply:
Good to know, though in this case I am choosing a new name in in this.setState(...) in the next line anyway?

Reply:
Yup, just some syntactic sugar for future use if you want.

Reply:
Thanks, there was another place where I needed that actually.